website/docs/pushdown.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 1
+---
+
+# Query pushdown
+
+**Query pushdown** is the method of optimizing a query by reducing the amount of data read and processed.
+It saves memory space, network bandwidth, and reduces the query execution time by not prefetching
+unnecessary data to the database management system.
+
+By default, FerretDB retrieves all data related to queried collection, and applies filters on its own, making
```

Comment:
```suggestion
Initially FerretDB retrieved all data related to queried collection, and applies filters on its own, making
```
I'm trying to make this sound like what we used to do and what we do now, instead of making it sounds like we still by default fetch all data.